src/vrapp/VRVolumeApp.cpp

```diff
@@ -1108,3 +1202,48 @@ void VRVolumeApp::do_grab(glm::mat4& newPose)
   m_controller_pose = newPose;
 }
 
+ArcBallCamera& VRVolumeApp::get_trackball_camera()
+{
+  return m_trackball;
+}
+
+void VRVolumeApp::set_animation_speed(float time)
+{
+  m_animation_speed = time;
+}
+
+Simulation& VRVolumeApp::get_simulation()
+{
+  return *m_simulation;
+}
+
+void VRVolumeApp::set_clip_min(glm::vec3 clip_min)
+{
+  m_ui_view->set_clip_min(clip_min);
+}
+
+void VRVolumeApp::set_clip_max(glm::vec3 clip_max)
+{
+  m_ui_view->set_clip_max(clip_max);
+}
+
+std::string VRVolumeApp::get_movie_state_label()
+{
+  if (m_current_movie_state == STOP)
+  {
+    return "Write Movie";
+  }
+  
+  return "STOP RECORD";
```

Comment:
I think the logic is a little easier to read if this is an if/else block

Reply:
I disagree.  But I changed it.

Reply:
How come? Either way is totally fine I'm just curious to hear your opinion

Reply:
Just coding styles. Because there is a return in the if statement, the compiler optimizer will ignore the branching `else`, so no extra code needs to be added. The same way on for loops is better to use pre-increment rather than post-increment operators. I added because you said, to make the logic easier to read.